pkgs/top-level/all-packages.nix

```diff
@@ -28229,9 +28229,11 @@ with pkgs;
 
   wrapQemuBinfmtP = callPackage ../applications/virtualization/qemu/binfmt-p-wrapper.nix { };
 
-  qgis-unwrapped = libsForQt5.callPackage ../applications/gis/qgis/unwrapped.nix {
-    withGrass = false;
-  };
+  qgis-ltr-unwrapped = libsForQt5.callPackage ../applications/gis/qgis/unwrapped-ltr.nix { };
```

Comment:
are the unwrapped versions ever expected to be used?

If not, I would like push them into the wrapped expressions. and just expose them as something like `qgis-litr.passthru.unwrapped` or something similar.

Reply:
Unless @mpickering can think of why they'd be used, I agree and can adjust accordingly.

Reply:
No I don't see why you would use the unwrapped versions.

Reply:
Something like this: https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/window-managers/qtile/default.nix

but you can lose the `buildEnv` logic.

Reply:
Updated with 3f6832d. Does that work?